cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -208,3 +228,95 @@ def test_collect_unconfigured(appliance):
+@pytest.mark.meta(blockers=[BZ(1436367, forced_streams=["5.8"])])
+@pytest.mark.parametrize('from_slave', [True, False], ids=['from_slave', 'from_master'])
+@pytest.mark.parametrize('zone_collect', [True, False], ids=['zone_collect', 'server_collect'])
+@pytest.mark.parametrize('collect_type', ['all', 'current'], ids=['collect_all', 'collect_current'])
+@pytest.mark.tier(3)
+def test_collect_multiple_servers(log_depot, temp_appliance_preconfig, depot_machine_ip, request,
+                                  configured_external_appliance, zone_collect, collect_type,
+                                  from_slave):
+
+    appliance = temp_appliance_preconfig
+    log_depot.machine_ip = depot_machine_ip
+
+    @request.addfinalizer
+    def _clear_ftp():
+        with log_depot.ftp as ftp:
+            ftp.cwd(ftp.upload_dir)
+            ftp.recursively_delete()
+
+    # Prepare empty workspace
+    with log_depot.ftp as ftp:
+        # move to upload folder
+        ftp.cwd(ftp.upload_dir)
+        # delete all files
+        ftp.recursively_delete()
+
+    with appliance:
+        uri = log_depot.machine_ip + log_depot.access_dir
+        depot = configure.ServerLogDepot(log_depot.protocol,
+                                         depot_name=fauxfactory.gen_alphanumeric(),
+                                         uri=uri,
+                                         username=log_depot.credentials["username"],
+                                         password=log_depot.credentials["password"],
+                                         second_server_collect=from_slave,
+                                         zone_collect=zone_collect
+                                         )
+        depot.create()
+
+        if collect_type == 'all':
+            depot.collect_all()
+        else:
+            depot.collect_current()
+
+    if from_slave and zone_collect:
+        check_ftp(log_depot.ftp, appliance.slave_server_name(), appliance.slave_server_zone_id())
+        check_ftp(log_depot.ftp, appliance.server_name(), appliance.server_zone_id())
+    elif from_slave:
+        check_ftp(log_depot.ftp, appliance.slave_server_name(), appliance.slave_server_zone_id())
+    else:
+        check_ftp(log_depot.ftp, appliance.server_name(), appliance.server_zone_id())
+
+
+@pytest.mark.meta(blockers=[BZ(1436367, forced_streams=["5.8"])])
+@pytest.mark.parametrize('zone_collect', [True, False], ids=['zone_collect', 'server_collect'])
+@pytest.mark.parametrize('collect_type', ['all', 'current'], ids=['collect_all', 'collect_current'])
+@pytest.mark.tier(3)
+def test_collect_single_servers(log_depot, appliance, depot_machine_ip, request, zone_collect,
+                                collect_type):
+    log_depot.machine_ip = depot_machine_ip
+
+    @request.addfinalizer
+    def _clear_ftp():
+        with log_depot.ftp as ftp:
+            ftp.cwd(ftp.upload_dir)
+            ftp.recursively_delete()
```

Comment:
@mfalesni @izapolsk 
Test should delete all files from ftp before and after tests, so if i will execute ftp.recursively_delete(d="/") it will raise exeption because it will try to delete "/" folder.
So current implementation deletes all files from ftp 
Note: "/" is a root directory for ftp, in our case it is equal to "/home/jenkins" and we creating ftp depot vm for this tests and after this, we deleting this vm.